### ISSUE TITLE
Legenda de vídeos

### ISSUE THREAD

**Issue by henricavalcante** [cerebrobr/cerebro#18]
Estou em contato com o pessoal do Amara.org na tentativa de criação de um time para legendagem de vídeos, quem se interessa inicialmente para colaborar?

**Comment by rodneidesign** [comment]
@henricavalcante seriam vídeos específico da área técnica?

**Comment by henricavalcante** [comment]
Sim, vamos tentar manter o foco ao máximo que é a disseminação do conteúdo técnico em desenvolvimento de software como está descrito no [README](https://github.com/cerebrobr/cerebro/blob/master/README.md) , tem muito conteúdo em vídeo que muita gente não tem acesso por não ter um certo nível de inglês.

**Comment by rodneidesign** [comment]
:+1:

**Comment by henricavalcante** [comment]
@rodneidesign seu (y) significa que quer participar da equipe?

**Comment by rodneidesign** [comment]
sim @henricavalcante e que também aprovo sua ideia  :)

**Comment by danilovaz** [comment]
muito bom :+1:

**Comment by nicholasess** [comment]
Aprovo a ideia, @cerebrobr/colaboradores @cerebrobr/editores, aprovam também?
@henricavalcante se não for incomodo, poderia já agilizar isso?
Ter pessoas para colaborar sem ter algo em andamento, não cria expectativas, acho interessante você ser o lider, e já começar a traduzir algo como exemplo pra galera.
Dessa maneira, temos uma justificativa para galera colaborar, ou seja, você já está fazendo isso e precisa de ajuda dos demais colaboradores.

**Comment by nicholasess** [comment]
@henricavalcante desculpe pela inocência, mas essa issue tinha sido referenciada a issue de sugestão, porém acredito que isso é uma necessidade de ser implantada. Pois somente artigos, não fazem a galera que é leiga em um assunto, entender por completo.

**Comment by henricavalcante** [comment]
Amanhã a noite terei um tempo para iniciar a tradução de um vídeo como forma de demonstração de como funciona a plataforma.

**Comment by henricavalcante** [comment]
http://amara.org/en/videos/KU4Gym7kilg6/info/exploring-the-arduino-with-nodejs-and-johnny-five-patrick-catanzariti/

**Comment by henricavalcante** [comment]
Acho que para organizar essa parte dos vídeos que estão sendo legendados e os pedidos de legendas deveriamos criar um repositório somente para os videos onde postaríamos o conteúdo em andamento e concluído e a galera poderia solicitar legendas pelas issues.

**Comment by henricavalcante** [comment]
@batusa

**Comment by batusa** [comment]
@henricavalcante assim que tiver o time no Amara, entrarei para ajudar na legenda e revisão ;)

**Comment by henricavalcante** [comment]
Eles estão querendo cobrar para a criação do time, mandei outro email falando que nossa instituição é sem fins lucrativos, vou aguardar para ver no que dá.
